src/site/content/en/blog/vitals-tools-workflow/index.md

```diff
@@ -0,0 +1,507 @@
+---
+title: A performance-focused workflow based on Google tools
+subhead: Combine Google tools to audit, improve and monitor your website effectively.
+authors:
+  - antoinebisch
+  - gmimani
+date: 2021-08-05
+hero: image/PTlEGTxNW2XkfI8mA3pIGDFxdYY2/1qMzdR1XGj3HKzuW4PFG.jpg
+thumbnail: image/PTlEGTxNW2XkfI8mA3pIGDFxdYY2/ApVe49OiB4KIJwcC7njy.jpg
+alt: An image of the various tools described in this article.
+description: >
+  With the growing importance of Core Web Vitals, site owners and developers increasingly focus on performance and key user experiences. Google provides many tools to help evaluate, optimize, and monitor pages, but users are often confused by the different sources of data and how to leverage them effectively. This article proposes a workflow combining several tools and clarifies where and how they make sense along the development process.
+tags:
+  - blog
+  - web-vitals
+  - performance
+---
+
+With the [Core Web Vitals](/vitals/) (CWV)
+metrics now incorporated into
+[Page experience signals](https://developers.google.com/search/docs/guides/page-experience#signals),
+user experience optimization has increasingly become top of mind for website owners.
+This article will suggest how the
+[various performance-focused tools](/vitals-tools/)
+that Google has made freely available can be combined to effectively evaluate
+website health and identify pain points,
+debug and optimize pages that need attention,
+and continuously monitor and prevent regression.
+
+<figure class="w-figure">
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/aQKhuF8mZNiQNrMXhpkm.png",
+alt="A diagram of the various steps outlined in this article. Evaluate website health and identify pain points, debug and optimize, monitor and continuous development.",
```

Comment:
```suggestion
alt="A diagram of the cycle outlined in this article: evaluate website health and identify pain points, debug and optimize, monitor and continuous development.",
```

Reply:
In the final section before the conclusion, you mention the cyclical nature of this process in text. If I can see, I get this message three times: twice in a diagram and once in text. If I'm visually impaired I only get it one time, in the text.  This section corrects this.